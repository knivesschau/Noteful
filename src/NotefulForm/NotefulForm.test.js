import React from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import NotefulForm from './NotefulForm';

describe ('NotefulForm component', () => {
    const props = {
        className: 'test-class-name',
        children: <p>test children</p>,
        'data-other': 'test-other-prop'
    };

    it ('renders a NotefulForm (form) by default', () => {
        const wrapper = shallow(<NotefulForm/>)
        expect(toJson(wrapper)).toMatchSnapshot()
    });

    it ('renders the NotefulForm given the props', () => {
        const wrapper = shallow(<NotefulForm {...props}/>)
        expect(toJson(wrapper)).toMatchSnapshot()
    });
})