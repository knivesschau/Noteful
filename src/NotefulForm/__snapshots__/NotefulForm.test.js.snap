// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotefulForm component renders a NotefulForm (form) by default 1`] = `
<form
  action="#"
  className="Noteful-form "
/>
`;

exports[`NotefulForm component renders the NotefulForm given the props 1`] = `
<form
  action="#"
  className="Noteful-form test-class-name"
  data-other="test-other-prop"
>
  <p>
    test children
  </p>
</form>
`;
